setup.py

```diff
@@ -70,7 +66,7 @@
     data_files=[
         ("share/man/man1", ["docs/rdiff-backup.1", "docs/rdiff-backup-statistics.1"]),
         (
-            "share/doc/rdiff-backup-%s" % (version_string,),
+            "share/doc/rdiff-backup",
```

Comment:
Rrr, that's pretty much standard to have the docs versioned under Linux. No trick to keep it that way?

Reply:
Before doing the change, I had a look at my `/usr/share/doc/` directory and I don't see any version in the directories name. In Debian, this folder must matches the packages name.

Reply:
You're right, can't remember where I saw this, Forget about it.